src/main/java/level4.java

```diff
@@ -1,2 +1,75 @@
-package PACKAGE_NAME;public class level4 {
+import java.util.Scanner;
+
+public class level4 {
+    public static void main(String[] args){
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        String line;
+        Task[] tasks = new Task[100];
+        boolean[] done = new boolean[100];
+        String[] type = new String[100];
+        int i = 0;
```

Comment:
The integer i is also not very clearly labelled, hence someone reading the code for the first time might not know what it is.